src/routes/webhooks/paddle.ts

```diff
@@ -682,41 +682,49 @@ export const processTransactionCompleted = async ({
         throw new Error('User does not have access to cores purchase');
       }
 
-      try {
-        await purchaseCores({
-          transaction: userTransaction,
-        });
-      } catch (error) {
-        if (error instanceof TransferError) {
-          const userTransactionError = new UserTransactionError({
-            status: error.transfer.status,
+      const shouldSkipNjord =
+        !!transactionData.discountId &&
+        !!remoteConfig.vars.paddleTestDiscountIds?.includes(
+          transactionData.discountId,
+        );
+
+      if (!shouldSkipNjord) {
+        try {
+          await purchaseCores({
             transaction: userTransaction,
           });
```

Comment:
Just skip njord call, so Cores are not assigned, everything else works so we can e2e test.

Reply:
@idoshamun thing better then messing with separate njord system account for this test case

Reply:
So the checkout should work but we will not see our balance gets updated?

Reply:
yes

Reply:
if team members need balance we can manually assign as needed

Reply:
ok, let's see if someone complains 😂

Reply:
I'll let people know